modules/creating-nfd-cr-cli-disconnected.adoc

```diff
@@ -0,0 +1,167 @@
+// Module included in the following assemblies:
+//
+// * hardware_enablement/psap-node-feature-discovery-operator.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="creating-nfd-cr-cli-disconnected_{context}"]
+= Create a NodeFeatureDiscovery CR by using the CLI in a disconnected environment
+
+As a cluster administrator, you can create a `NodeFeatureDiscovery` CR instance using the {oc-first}.
```

Comment:
```suggestion
As a cluster administrator, you can create a `NodeFeatureDiscovery` CR instance by using the {oc-first}.
```
From the _using_ entry in IBM Style word usage: "To avoid ambiguity, replace this gerund with either _by using_ or _that uses_."